sdk/src/main/scala/io/horizen/SidechainSettings.scala

```diff
@@ -26,8 +26,9 @@ case class WebSocketClientSettings(
 
 case class WebSocketServerSettings(
     wsServer: Boolean = false,
-    wsServerPort: Int = 8025
-) extends SensitiveStringer
+    wsServerPort: Int = 8025,
+    wsServerAllowedOrigins: Seq[String] = Seq("127.0.0.1")
```

Comment:
are we sure about that default ipv4 localhost is enough?
what if the server has only ipv6 configuration or other loopback operations?

maybe we may make default value as `Seq()` and inside `WebSocketAccountConfigurator` class detect this case. Then check it in a more comprehensive way.
For example use InetAddress class methods: `address.isSiteLocalAddress || address.isLoopbackAddress || address.isAnyLocalAddress`